CHANGES.md

```diff
@@ -142,6 +100,33 @@ Updates to Chapel's Release Formats
   - use `libunwind` to enable stack traces
 * added a Fedora 43 Linux package and retired the Fedora 41 package
 
+Language Feature Improvements
+-----------------------------
+* added support for `.domain` queries on array types, as with array values  
+  (e.g., `proc foo(type t) { ...t.domain... }  foo([1..3] real);` now works)
+* added support for promoting `min()`/`max()` over general iterable exprs  
+  (e.g., `min([i in 1..4] i, [4, 3, 2, 1])` now works)
+
+Semantic Changes / Changes to the Language Definition
+-----------------------------------------------------
+* `manage` statements in non-throwing contexts can now contain throwing code
+
+Performance Optimizations / Improvements
+----------------------------------------
+* removed checks in `integral.safeCast(bool)` when compiling with `--no-checks`
+
+Deprecated / Unstable / Removed Language Features
+-------------------------------------------------
+* removed the deprecated `IO.ioendian` type
+* removed `Sort.sort()` and `.isSorted()` overloads with deprecated arguments
+* remove the deprecated `Python.importModule` method
```

Comment:
Minor inconsistency: the other three say "removed" (and I think most other places use past tense) :)